pkg/build/api/types.go

```diff
@@ -266,6 +266,18 @@ const (
 	// secret for pushing the output image.
 	// The build will stay in the pending state until the secret is created, or the build times out.
 	StatusReasonMissingPushSecret = "MissingPushSecret"
+
+	// StatusReasonFailedPostCommitHook indicates that the postcommit failed
+	StatusReasonPostCommitHookFailed = "PostCommitHookFailed"
+
+	// StatusReasonGitCloneFailed
+	StatusReasonGitCloneFailed = "GitCloneFailed"
+
+	// StatusReasonPullImageFailed
+	StatusReasonPullImageFailed = "PullImageFailed"
+
+	// StatusReasonRunAssembleContainer
+	StatusReasonRunAssembleContainer = "AssembleContainerFailed"
 )
```

Comment:
we're explicitly invoking commit on the container, so i'd assume so... (for s2i builds. not for docker builds, of course)